### ISSUE TITLE
Music volume is not set correctly

### ISSUE THREAD

**Issue by AXDOOMER** [nukeykt/PCDoom-v2#14]
When playing with the [release](https://github.com/nukeykt/PCDoom-v2/releases) executable, I noticed that the music plays at the maximum volume although I set the music volume lower. 

Here's my `DEFAULT.CFG`, you can see my SFX volume and music volume are very low. I tried with GUS and General Midi and I get the same bug. If I move the music slider in the F4 menu, then the music becomes played at the supposed volume. Maybe it's an initialisation problem which always set the music volume to an arbitrary value? 
```
mouse_sensitivity		9
sfx_volume		2
music_volume		5
show_messages		1
key_right		77
key_left		75
key_up		17
key_down		31
key_strafeleft		30
key_straferight		32
key_fire		29
key_use		57
key_strafe		56
key_speed		54
use_mouse		1
mouseb_fire		0
mouseb_strafe		1
mouseb_forward		2
use_joystick		0
joyb_fire		0
joyb_strafe		1
joyb_use		3
joyb_speed		2
screenblocks		10
detaillevel		0
showmessages		1
comport		1
snd_channels		16
snd_musicdevice		5
snd_sfxdevice		3
snd_sbport		544
snd_sbirq		7
snd_sbdma		1
snd_mport		-1
usegamma		0
chatmacro0		"no macro"
chatmacro1		"no macro"
chatmacro2		"no macro"
chatmacro3		"no macro"
chatmacro4		"no macro"
chatmacro5		"no macro"
chatmacro6		"no macro"
chatmacro7		"no macro"
chatmacro8		"no macro"
chatmacro9		"no macro"
		0
```

**Comment by nukeykt** [comment]
This bug already was fixed in 8ce5f2a

**Comment by AXDOOMER** [comment]
Can you please update your release? Thanks. 

https://github.com/nukeykt/PCDoom-v2/releases
